scripts/var_props.py

```diff
@@ -928,12 +941,9 @@ def __init__(self, var1_stdname, var1_type, var1_kind, var1_units,
         # end if
         if self.__compat:
             # Check dimensions
-            ##XXgoldyXX: For now, we always have to create a dimension
-            ##           transform because we do not know if the vertical
-            ##           dimension is flipped.
             if var1_dims or var2_dims:
                 _, vdim_ind = find_vertical_dimension(var1_dims)
-                if (var1_dims != var2_dims) or (vdim_ind >= 0):
+                if (var1_dims != var2_dims):
```

Comment:
I guess we have not needed that `or` since `var[12]_top` was brought in. But that makes me wonder what is going on with line 938. That line is redundant, is it supposed to be something else (e.g., `self.__equiv = False`)?

Reply:
@dustinswales do you have insight on this. It looks like "self.__compat = True" is happening within the "if self.__compat" block.

Reply:
We can follow up on this after merging the PR, leaving the comment as unresolved.